### DISCUSSION TITLE
Good clean up change to S. 6(7)

### DISCUSSION THREAD

**Discussion by Aaron-XReg** [MinistryFinancialServices/Virtual-Asset-Amendment-Consultation#28] [General]
Glad to see that section 6(7) is proposed for amendment. It currently states that a registered person shall not publicly state, imply or convey that it is regulated in the Islands which was always very confusing. Under the new proposed drafting the point is clear that a registrant should not claim to be licensee which makes a lot more sense.

**Comment by CaraProvenance** [comment]
Equally glad to see this has been specifically addressed as there has been some confusion with regards the expression of registrations as licenses.

Would suggest the category for VASP License and Sandbox License should also be added as additional categories under https://www.cima.ky/search-entities-cima to facilitate transparency and ease of search for the # of entities approved as registrants, licensees, sandbox licensees.
